### PR TITLE
Added notes about PyCon and EuroPython web assembly summits.

### LINKED ISSUE #6: Missing notes from PyCon US 2024
Are there any updates to these meetings - especially for WASI 2 support

### PR DESCRIPTION
**Pull request by ntoll**
Fixes #6

NB: These should have been written up sooner (my apologies). But life, the universe and everything got in the way. This may explain their rather sketchy nature. I've included appropriate links where possible.

### CODE REVIEW

File: meetings/2024/webassembly-summit-europython.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# EuroPython 2024 Web Assembly Summit
+
+Held in Prague. Event website here: https://ep2024.europython.eu/session/webassembly-summit 
+
+We had around 15 sign up, and several more turn up on the day.
+
+As last yeat, the morning consisted of presentations, with a discussion
```
Comment (mahmoud) [inline meetings/2024/webassembly-summit-europython.md]:
```suggestion
As with last year, the morning consisted of presentations, with a discussion
```

File: meetings/2024/webassembly-summit-europython.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            * **TODO**: we should create an FFI-aware kernel to run games (like PGZero).
+            * Check out [Streamlit](https://streamlit.io/).
+        * PyScript [LTK](https://github.com/pyscript/ltk) - client side rendering of UIs on top of jQuery and PyScript.
+        * Integration with JS frameworks:
+            * [HTMXXX](https://htmxxx.fly.dev/)
+            * Jinja templates to JS?
+* Pyodide focused:
+    * Reducing the footprint, compression - do like Pyrun (remove stuff, compress).
+    * Reducing startup time:
+        * We need to improve packages to download all dependencies at once, rather than one wheel at a time.
```
Comment (mahmoud) [inline meetings/2024/webassembly-summit-europython.md]: Big +1 here. I commented on an issue around this a while back, might be worth linking? https://github.com/pyodide/pyodide/issues/2045#issuecomment-1423248534

File: meetings/2024/webassembly-summit-pycon-us.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# WebAssembly Summit at PyCon US 2024 - Pittsburgh
+
+[Website](https://us.pycon.org/2024/events/webassembly-summit/)
+
+Organisers:
+
+* Dr.Brett Cannon - Microsoft
```
Comment (mahmoud) [inline meetings/2024/webassembly-summit-pycon-us.md]:
```suggestion
* Dr. Brett Cannon - Microsoft
```

File: meetings/2024/webassembly-summit-pycon-us.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Fabio Pliger - Anaconda
+* Nicholas Tollervey - Anaconda
+
+## Agenda
+
+### Morning - talks MC'd by Nicholas
+
+* Pyodide (Dr.Hood Chatham) - An update on the current state of Pyodide (CPython compiled to WASM) by Hood, a core maintainer of the project.
```
Comment (mahmoud) [inline meetings/2024/webassembly-summit-pycon-us.md]:
```suggestion
* Pyodide (Dr. Hood Chatham) - An update on the current state of Pyodide (CPython compiled to WASM) by Hood, a core maintainer of the project.
```

File: meetings/2024/webassembly-summit-pycon-us.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    * [Blog post with updates](https://blog.pyodide.org/posts/0.26-release/)
+* MicroPython (Dr.Damien George) - A pre-recorded video updating us on the current state of MicroPython compiled to web assembly.
```
Comment (mahmoud) [inline meetings/2024/webassembly-summit-pycon-us.md]:
```suggestion
* MicroPython (Dr. Damien George) - A pre-recorded video updating us on the current state of MicroPython compiled to web assembly.
```

File: meetings/2024/webassembly-summit-pycon-us.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    * [Unlisted YouTube version](https://youtu.be/6icY5Pq3NOM)
+* WASI and Python (Eric Snow/Dawn Wages/Mike Droettboom) - An outline of the current state of WASI (WebAssembly System Interface) support in CPython.
+    * [Tier 2 support is here](https://discuss.python.org/t/wasi-has-been-promoted-to-a-tier-2-platform/45525)
+* PyScript (Fabio Pliger) - Updates, developments, future plans in the world of PyScript (a platform for Python in the browser).
+    * [PyScript homepage](https://pyscript.net)
+* PySheets / LTK (Dr.Chris Laffra) - a demonstration and description of PySheets - a spreadsheet UI for notebook like coding, written with PyScript.
```
Comment (mahmoud) [inline meetings/2024/webassembly-summit-pycon-us.md]:
```suggestion
* PySheets / LTK (Dr. Chris Laffra) - a demonstration and description of PySheets - a spreadsheet UI for notebook like coding, written with PyScript.
```

**Review by mahmoud — COMMENTED** [review]
A comment + some typos/nits. Thanks for this!

File: meetings/2024/webassembly-summit-europython.md (hunk 2 above)
Comment (ntoll) [reply meetings/2024/webassembly-summit-europython.md]: ...and updated.

**Comment by ntoll** [comment]
@mahmoud OK... so I've just spent 15mins Googling around use of space, titular abbreviations and period together. :wink:

At least we're not arguing over the use (or not) of the Oxford comma. :stuck_out_tongue:

**Comment by mahmoud** [comment]
> @mahmoud OK... so I've just spent 15mins Googling around use of space, titular abbreviations and period together. 😉
> 
> At least we're not arguing over the use (or not) of the Oxford comma. 😛

lol yes, the first one I thought was a typo, but increasingly I was worried it was a valid convention was just updating them for consistency :sweat_smile: Glad it wasn't in vain!

**Review by willingc — APPROVED** [review]
Thanks @ntoll. I'm going to merge this.

**Pull request merged**